src/aspect.ts

```diff
@@ -27,7 +29,7 @@ export abstract class ECRRepositoryAspect implements IAspect {
   /**
    * @internal
    */
-  static readonly _repoPolicies = new Map<string, Policy>();
+  static readonly _repoPolicies = new Map < string, Policy > ();
```

Comment:
I prefer not to add a space in between. i.e. keep `static readonly _repoPolicies = new Map<string, Policy>();`